mobile-app/app/screens/AppNavigator/screens/Balances/components/Announcements.tsx

```diff
@@ -24,14 +26,58 @@ export function Announcements (): JSX.Element {
     hiddenAnnouncements,
     hideAnnouncement
   } = useDisplayAnnouncement()
+  const isBlockchainDown = useBlockchainStatus()
+  const deFiChainStatusUrl = 'https://status.defichain.com/'
 
+  const blockChainIsDownContent: AnnouncementData[] = [{
+    lang: {
+      en: 'We are currently investigating a syncing issue on the blockchain. View more details on the DeFiChain Status Page.',
+      de: 'Wir untersuchen derzeit ein Synchronisierungsproblem der Blockchain. Weitere Details auf der DeFiChain Statusseite.',
+      'zh-Hans': '我们目前正在调查区块链上的同步化问题。前往 DeFiChain Status 页面 了解更多状态详情。',
+      'zh-Hant': '我們目前正在調查區塊鏈上的同步化問題。前往 DeFiChain Status 頁面 了解更多狀態詳情。',
+      fr: '' // get translation from team
+    },
+    version: '0.0.0',
+    url: {
+      ios: deFiChainStatusUrl,
+      android: deFiChainStatusUrl,
+      windows: deFiChainStatusUrl,
+      web: deFiChainStatusUrl,
+      macos: deFiChainStatusUrl
+    }
+  }]
+
+  const [emergencyMsgContent, setemergencyMsgContent] = useState<AnnouncementData[] | undefined>()
   const announcement = findAnnouncementForVersion(nativeApplicationVersion ?? '0.0.0', language, announcements)
-  const displayAnnouncement = getDisplayAnnouncement(hiddenAnnouncements, announcement)
+  const emergencyAnnouncement = findAnnouncementForVersion('0.0.0', language, emergencyMsgContent)
+  const existingAnnouncements = getDisplayAnnouncement(hiddenAnnouncements, announcement)
```

Comment:
If the blockchain is `down` and the `existingAnnouncements` is false, it won't show the warning. You have to pass the emergencyAnnouncement if it exists here. I believe the emergency announcement should take precedence in terms of displaying it to the user. 

![image](https://user-images.githubusercontent.com/13292662/151274117-3b0e0d84-3fa0-448d-a323-e3f22184e669.png)